visualizations/frontend/other/TTable.vue

```diff
@@ -1067,11 +1108,35 @@ export default {
     setPagerResetFunction(resetFunction) {
       this.pagerResetFunction = resetFunction;
     },
+    updateFilteredColumns(filterableColumnsToShow) {
+      this.filterableColumnsToShow = filterableColumnsToShow;
+      const thisTable = this.$store.state.layers.components[this.tag_unique];
+      let allValidColumns = [];
+      filterableColumnsToShow.forEach((fcts) => {
+        allValidColumns = allValidColumns.concat(fcts.sqlColumns);
+      });
+      // remove duplicate entries
+      allValidColumns = [...new Set(allValidColumns)];
+      if(allValidColumns.includes(undefined)){
+        console.error('Modify Column Configuration includes "undefined" value, please check that all sql column names are quoted!')
+      }
+      thisTable.filters.column_list = JSON.stringify(allValidColumns);
```

Comment:
This is breaking store immutability. We should create a layer mutation to add `column_list` to filters or you should set the whole `filters` object extending it with what you need to add (column_list) using the `storeAttribute` mutation from the layer store.